open_schools_platform/organization_management/teachers/services.py

```diff
@@ -0,0 +1,84 @@
+import uuid
+
+from phonenumber_field.phonenumber import PhoneNumber
+from rest_framework.exceptions import NotAcceptable, MethodNotAllowed
+
+from open_schools_platform.common.services import model_update, BaseQueryHandler
+from open_schools_platform.common.utils import filter_dict_from_none_values
+from open_schools_platform.organization_management.circles.models import Circle
+from open_schools_platform.photo_management.photos.services import create_photo
+from open_schools_platform.query_management.queries.models import Query
+from open_schools_platform.query_management.queries.services import query_update
+from open_schools_platform.organization_management.teachers.models import TeacherProfile, Teacher
+from open_schools_platform.user_management.users.models import User
+
+
+def create_teacher_profile(name: str, age: int = None, user: User = None,
```

Comment:
Thing about creating teacher profile, do we need to create it automatically, when we are creating user (as other profiles)? If we do need that, you are going to need to add create_teacher_profile to create_user service.